mp4parse/src/lib.rs

```diff
@@ -1176,12 +1192,11 @@ fn read_video_desc<T: Read>(src: &mut BMFFBox<T>, track: &mut Track) -> Result<S
 /// Parse an audio description inside an stsd box.
 fn read_audio_desc<T: Read>(src: &mut BMFFBox<T>, track: &mut Track) -> Result<SampleEntry> {
     let name = src.get_header().name;
-    track.mime_type = match name {
+    track.codec_type = match name {
         // TODO(kinetik): stagefright inspects ESDS to detect MP3 (audio/mpeg).
-        BoxType::MP4AudioSampleEntry => String::from("audio/mp4a-latm"),
-        // TODO(kinetik): stagefright doesn't have a MIME mapping for this, revisit.
-        BoxType::OpusSampleEntry => String::from("audio/opus"),
-        BoxType::ProtectedAudioSampleEntry => String::from("audio/crypto"),
+        BoxType::MP4AudioSampleEntry => CodecType::AAC,
+        BoxType::OpusSampleEntry => CodecType::OPUS,
+        BoxType::ProtectedAudioSampleEntry => CodecType::EncyptedAudio,
         _ => return Err(Error::Unsupported("unhandled audio sample entry type")),
```

Comment:
I think you can set codec_type to Unknown here now rather than returning an error